model/message_export.go

```diff
@@ -0,0 +1,53 @@
+package model
```

Comment:
Missing:

```
// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
// See License.txt for license information.
```

Reply:
Fixed in https://github.com/mattermost/mattermost-server/pull/7612/commits/16d8a4ca619697f61f84211bb38780fd8527be64